build.properties

```diff
@@ -44,3 +44,4 @@ jmeter.run.port=8080
 
 izpack.version=5.0.7-RC1
 pentaho-eula-wrap-config.version=8.0-SNAPSHOT
+dependency.kafka-clients.version=0.10.2.1
```

Comment:
@kurtwalker moved the line to more appropriate place. Other dependencies uses "dependency.xxx.version" pattern, so I used the same.